core/ui/public/i18n/en/translation.json

```diff
@@ -938,7 +939,10 @@
         "update_to_stable_version": "Update to stable version",
         "update_to_testing_version": "Update to testing version",
         "num_instances_installed": "{num} instance installed | {num} instances installed",
-        "reason_max_per_node_limit": "Limit of {param} instance reached | Limit of {param} instances reached"
+        "reason_max_per_node_limit": "Limit of {param} instance reached | Limit of {param} instances reached",
+        "core_app": "Core apps",
+        "search_core_module": "Search core apps",
+        "core_app_are_going_to_be_updated": "Core will be updated in a moment"
```

Comment:
I'd remove this string and reuse `software_center.core_update_will_start_in_a_moment`